oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -131,6 +131,22 @@ telemetry:
     port: ${SW_TELEMETRY_PROMETHEUS_PORT:1234}
 configuration:
   none:
+#  nacos:
+#    # Nacos Server Host
+#    serverAddr: 127.0.0.1
+#    # Nacos Server Port
+#    port: 8848
+#    # Nacos Configuration Group
+#    group: 'skywalking'
+#    # Dynamically configured keys
+#    dataIds:
+#      - receiver-trace.default.slowDBAccessThreshold
```

Comment:
Don't like this config. But the register should work automatically.

Reply:
Rebased to the refactored code and this is no more needed